WebContent/zip-ext.js

```diff
@@ -208,11 +208,165 @@
 	FileWriter.prototype = new Writer();
 	FileWriter.prototype.constructor = FileWriter;
 
+	function DBWriter(contentType) {
+		var db, tempDB, that = this, blobs, dbName = "zipjs", instance;
+
+		function init(callback, onerror) {
+			var request = indexedDB.open(dbName, 5);
+			request.onerror = onerror;
+			request.onupgradeneeded = function (event) {
+				db = event.target.result;
+				db.createObjectStore("instances", { autoIncrement: true });
+			};
+			request.onsuccess = function (event) {
+				db = event.target.result;
+				addInstance(callback, onerror);
+			};
+			blobs = [];
+		}
+
+		function addInstance(callback, onerror) {
```

Comment:
The use of similarly named variables `addInstance` / `addInstanceDB` / `"instances"` is a bit confusing. It'd be nice if you replace the names with something that reveals a bit more about its intended/actual purpose.